skbio/stats/composition.py

```diff
@@ -2059,7 +2036,8 @@ def dirmult_ttest(
 
     # multiple comparison
     if p_adjust is not None:
```

Comment:
I think that you `_calc_p_adjust` function already takes care of `None` and upper case. Therefore these checks are not necessary. Same for the other instance where `_calc_p_adjust` is called.

Reply:
Thank you for your feedback! I have reviewed the function and removed the redundant code.